eng/common/scripts/Package-Properties.ps1

```diff
@@ -168,7 +168,7 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
         $lookup[$lookupKey] = $pkg
 
         foreach ($file in $targetedFiles) {
-            $filePath = Resolve-Path (Join-Path $RepoRoot $file)
+            $filePath = (Join-Path $RepoRoot $file)
```

Comment:
Because `AllPkgProperties` is populated in this run, we do not need to worry about path separators being cross-platform different.